tests/e2e/specs/rc-1290-acceptance.spec.ts

```
@@ -0,0 +1,428 @@
+      wpEval('update_option("faz_cookie_definitions_meta", array("updated_at"=>"2099-01-01 00:00:00","count"=>1,"source"=>"stale"), false);');
+      const newer = JSON.parse(
+        wpEval('$d = new FazCookie\\Includes\\Cookie_Definitions(); echo wp_json_encode( $d->get_meta() );')
+      );
+      expect(newer.source).not.toBe('bundled');
+    } finally {
+      // Independently: a throw in the first must not skip the second.
+      const errs: unknown[] = [];
+      for (const restore of [restoreDefs, restoreMeta]) {
+        try { restore(); } catch (e) { errs.push(e); }
+      }
+      if (errs.length) throw errs[0];
```

Biome (2.5.8)

[error] Unsafe usage of 'throw'.

(lint/correctness/noUnsafeFinally)